samples/drivers/mspi/mspi_flash/boards/apollo4p_evb.overlay

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) 2024 Ambiq Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	aliases {
+		flash0 = &atxp032;
+	};
+};
+
+&gpio64_95 {
+	status = "okay";
+};
+
+&gpio32_63 {
+	status = "okay";
+};
+
+&mspi1 {
+	pinctrl-0 = <&mspi1_default>;
+	pinctrl-1 = <&mspi1_sleep>;
+	pinctrl-2 = <&mspi1_flash>;
+	pinctrl-names = "default","sleep","flash";
+	status = "okay";
+
+	ce-gpios = <&gpio64_95 25 GPIO_ACTIVE_LOW>,
+				<&gpio64_95 23 GPIO_ACTIVE_LOW>;
```

Comment:
why is there two ce-gpios? plus the ce is 45

Reply:
I can remove the fact that there are 2 but the CE for the apollo4p_evb_disp_shield_rev1 is 89.
MSPI1_CE_GP89 is from the schematic. The CE works as is.